src/Symfony/Component/Config/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+5.1.0
+-----
+ * deprecated signature `NodeDefinition::setDeprecated(string $message)`, use `Definition::setDeprecation(string $package, string $version, string $message)` instead
+ * deprecated signature `BaseNode::setDeprecated(?string $message)`, use `Alias::setDeprecation(string $package, string $version, string $message)` instead
+ * deprecated passing a null message to `BaseNode::setDeprecated()` to un-deprecate a node is deprecated
```

Comment:
`deprecated ... is deprecated` (double "deprecated"?)

Reply:
nice catch